.github/workflows/ci.yaml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  merge_group:
+    branches: [ main ]
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  test_unit:
+    strategy:
+      matrix:
+        python-version: ["3.9","3.10","3.11"]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: make install-test
+    - name: Run unit tests
+      run: |
+        pip install  .
+        make test
```

Comment:
When https://github.com/Unstructured-IO/unstructured-python-client/pull/41 is merged, the `make test` will run also integration tests, which means docker. It would be good if that worked here just like that, but if not, then running `make test-unit` will be simple solution.

Reply:
I can make that change once it's merged, but #41, which adds `make test-unit`, also hasn't been merged so it doesn't make sense to update this yet.

Reply:
That's right. I thought it will be merged faster, but this one's ready, so accepting :)